src/get-started/test-service.md

```diff
@@ -59,22 +59,16 @@ contain the tests for transaction business logic.
 
 ### Imports
 
-The created file is executed separately from the service code, meaning that we
-need to import the service crate along with **exonum** and **exonum-testkit**:
-
-```rust
-extern crate exonum;
-extern crate exonum_cryptocurrency as cryptocurrency;
-#[macro_use] extern crate exonum_testkit;
-```
-
 Just like with the service itself, we then import the types we will use:
```

Comment:
IMO, this sentence could be improved after edits. I'd suggest to change to "We need to import the types we will use."